cmd/kops/get.go

```diff
@@ -87,6 +114,102 @@ func NewCmdGet(f *util.Factory, out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func RunGet(context Factory, out io.Writer, options *GetOptions) error {
+
+	client, err := context.Clientset()
+	if err != nil {
+		return err
+	}
+
+	cluster, err := client.Clusters().Get(options.clusterName)
+	if err != nil {
+		return err
+	}
+
+	if cluster == nil {
+		fmt.Fprintf(os.Stderr, "No cluster found\n")
+		return nil
+	}
+
+	clusterList := &api.ClusterList{}
+	clusterList.Items = make([]api.Cluster, 1)
+	clusterList.Items[0] = *cluster
+
+	args := make([]string, 0)
+
+	clusters, err := buildClusters(args, clusterList)
+
+	ig, err := client.InstanceGroups(options.clusterName).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	if ig == nil || ig.Items == nil || len(ig.Items) == 0 {
+		fmt.Fprintf(os.Stderr, "No instance groups found\n")
+	}
+
+	instancegroups, err := buildInstanceGroups(args, ig)
+	if err != nil {
+		return err
+	}
+
+	switch options.output {
+	case OutputYaml:
+
+		err = clusterOutputYAML(clusters, out)
+		if err != nil {
+			return err
+		}
+
+		if err := writeYAMLSep(out); err != nil {
+			return err
+		}
+
+		err = igOutputYAML(instancegroups, out)
+		if err != nil {
+			return err
+		}
+
+	case OutputJSON:
+		return fmt.Errorf("not implemented")
```

Comment:
Looking at this more, our API does not support this.  For JSON it has to be multiple files because we do not have a top level struct that includes both an ig and a cluster.  Because of that, we cannot output one JSON file to stdout.